cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -16,15 +18,17 @@ type Supervisor struct {
 
 // Capabilities is the set of capabilities that the Supervisor supports.
 type Capabilities struct {
-	AcceptsRemoteConfig    *bool `mapstructure:"accepts_remote_config"`
-	ReportsEffectiveConfig *bool `mapstructure:"reports_effective_config"`
-	ReportsOwnMetrics      *bool `mapstructure:"reports_own_metrics"`
-	ReportsHealth          *bool `mapstructure:"reports_health"`
-	ReportsRemoteConfig    *bool `mapstructure:"reports_remote_config"`
+	AcceptsRemoteConfig            *bool `mapstructure:"accepts_remote_config"`
```

Comment:
Sorry I had missed this in earlier, and isn't strictly relevant for the PR but I am curious, what does a pointer value add here?

Reply:
Since the zero-value for `bool` is `false` but the default value may be `true`, using a pointer instead allows us to use a nil pointer value as a way to represent when the user didn't pass a value in and a default should be used.